src/components/Navigation/Navigation.jsx

```diff
@@ -45,6 +45,15 @@ export default function Navigation({ children }) {
         >
           Transactions
         </NavLink>
+
+        <NavLink
+          className="NavigationLink"
+          to="/transaction-stats"
+          activeClassName="NavigationLinkActive"
+        >
+          Transaction Statistics & Reports
```

Comment:
How about just "Statistics" or just "Reports"?

Minor nit

Reply:
Changed to "Statistics"